packages/rrweb-snapshot/src/utils.ts

```diff
@@ -120,7 +120,7 @@ export function stringifyStylesheet(s: CSSStyleSheet): string | null {
     let sheetHref = s.href;
     if (!sheetHref && s.ownerNode && s.ownerNode.ownerDocument) {
       // an inline <style> element
-      sheetHref = s.ownerNode.ownerDocument.location.href;
+      sheetHref = s.ownerNode.ownerDocument.baseURI;
```

Comment:
```suggestion
      sheetHref = s.ownerNode.baseURI;
```

I don't think the ownerDocument bit is necessary anymore!